lib/drivers/medtronic/medtronicDriver.js

```diff
@@ -25,6 +25,7 @@ var common = require('../../commonFunctions');
 var medtronicSimulator = require('./medtronicSimulator');
 var crcCalculator = require('../../crc.js');
 var debugMode = require('../../../app/utils/debugMode');
+var api = require('../../core/api.js');
```

Comment:
@gniezen do we need this? Isn't this the same as cfg.api?

Reply:
Yep, you're right, I'll remove it.